### PR TITLE
Added a new formal jacket to the captain's locker

### PR DESCRIPTION
**Pull request by Thunder12345**
Replaces the spare carapace in the captain's locker, which it is mechanically identical to
learn to use github flavoured markdown
no seriously this is annoying me

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
looks liked a recolored Warden's jacket.

**Comment by Chocobro** [comment]
looks extremely sgt. pepper I love it :+1:

**Comment by Supermichael777** [comment]
Why do people have such a aversion to sub-typing these so the armor values are identical.

**Comment by Thunder12345** [comment]
There's no point making it a subtype of any of the other cap's gear, I'd just have to change more vars if I did it that way. Also I don't think it really needs a hat, I've never seen this type of uniform worn with a hat before.

**Comment by SconesC** [comment]
It actually looks really nice :+1:

File: code/modules/clothing/suits/jobs.dm [hunk 1]
```diff
@@ -22,6 +22,14 @@
 	flags_inv = HIDEJUMPSUIT
 	allowed = list(/obj/item/weapon/disk, /obj/item/weapon/stamp, /obj/item/weapon/reagent_containers/food/drinks/flask, /obj/item/weapon/melee, /obj/item/weapon/storage/lockbox/medal, /obj/item/device/assembly/flash/handheld, /obj/item/weapon/storage/box/matches, /obj/item/weapon/lighter, /obj/item/clothing/mask/cigarette, /obj/item/weapon/storage/fancy/cigarettes, /obj/item/weapon/tank/internals/emergency_oxygen)
 
+/obj/item/clothing/suit/capformal
```
Comment (GunHog) [inline code/modules/clothing/suits/jobs.dm]: Why not make it a child of the captunic? It will cut down on the copy-paste.

Comment (Thunder12345) [inline code/modules/clothing/suits/jobs.dm]: That just means I have to take away the HIDEJUMPSUIT flag, so I didn't see one way as being any better than the other

**Comment by Shapsy** [comment]
Why make it unarmored? Why not have it replace the spare carapace?

**Comment by Thunder12345** [comment]
Huh, never realised there's a second carapace. Would anyone complain if I did what Shapsy suggested?

**Comment by WJohn** [comment]
:+1: 
I really like this, great sprite.

**Pull request merged**